tests/unittests/Foundation/ReferenceFoundation/TestNSBundle.mm

```diff
@@ -116,18 +115,14 @@ virtual void SetUp() {
 
         NSError* error = nil;
         [[NSFileManager defaultManager] createDirectoryAtPath:tempDir withIntermediateDirectories:false attributes:nil error:&error];
-        if (error) {
-            TraceError(L"TestNSBundle", L"%hs", [[error description] UTF8String]);
-            return;
-        }
+        // WINOBJC: Change to ASSERT macro to remove LoggingNative reference
```

Comment:
fwiw i don't think this needs commenting, as the reference tests didn't have `TraceError` to remove to begin with